riff-cli/cmd/apply.go

```diff
@@ -92,13 +93,12 @@ func apply(opts ApplyOptions, kubectlClient kubectl.KubeCtl) error {
 		message = fmt.Sprintf("Applying resource %v\n\n", opts.FilePath)
 		cmdArgs = append(cmdArgs, "-f", abs)
 	}
-
+	if len(resourceDefinitionPaths) == 0 {
+		fmt.Printf("No function resources found in %v\n", opts.FilePath)
```

Comment:
this statement is true, but potentially misleading. There were also no topic resources found in the directory.

Perhaps say `No resources found in %v\n `? Or maybe `No riff resources found...`?